timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -42,211 +42,58 @@ limitations under the License.
-              {{ selectedScenario.description }}
-            </div>
-          </div>
-          <v-divider></v-divider>
-          <v-card-actions>
-            <v-spacer></v-spacer>
-            <v-btn @click="scenarioDialog = false" text> Cancel </v-btn>
-            <v-btn text color="primary" :disabled="!selectedScenario" @click="addScenario(selectedScenario.id)">
-              Add
-            </v-btn>
-          </v-card-actions>
-        </v-card>
-      </v-dialog>
-
-      <v-tabs v-model="leftPanelTab" grow>
-        <v-tab v-for="item in leftPanelTabItems" :key="item"> {{ item }} </v-tab>
-      </v-tabs>
-      <v-divider></v-divider>
-      <v-tabs-items v-model="leftPanelTab">
-        <v-tab-item :transition="false">
-          <ts-saved-searches v-if="meta.views"></ts-saved-searches>
-          <ts-data-types></ts-data-types>
-          <ts-tags></ts-tags>
-          <ts-graphs></ts-graphs>
-          <ts-stories></ts-stories>
-          <ts-intelligence></ts-intelligence>
-          <ts-search-templates></ts-search-templates>
-          <ts-sigma-rules></ts-sigma-rules>
-          <ts-analyzer-results></ts-analyzer-results>
-        </v-tab-item>
-        <v-tab-item :transition="false">
-          <ts-scenario v-for="scenario in activeScenarios" :key="scenario.id" :scenario="scenario"></ts-scenario>
-          <v-row class="mt-0 px-2" flat>
-            <v-col cols="6">
-              <v-card v-if="!Object.keys(scenarioTemplates).length" flat class="pa-4"
-                >No scenarios available yet. Contact your server admin to add scenarios to this server.</v-card
-              >
-              <v-btn v-else text color="primary" @click="scenarioDialog = true" style="cursor: pointer"
-                ><v-icon left>mdi-plus</v-icon> Add Scenario</v-btn
-              >
-            </v-col>
-
-            <v-col cols="6">
-              <v-btn
-                small
-                text
-                color="primary"
-                v-if="hiddenScenarios.length"
-                @click="showHidden = !showHidden"
-                class="mt-1"
-              >
-                <small
-                  ><span v-if="showHidden">Hide</span><span v-else>Show</span> hidden scenarios ({{
-                    hiddenScenarios.length
-                  }})</small
-                >
-              </v-btn>
-            </v-col>
-          </v-row>
-
-          <div v-if="showHidden">
-            <ts-scenario v-for="scenario in hiddenScenarios" :key="scenario.id" :scenario="scenario"></ts-scenario>
-          </div>
-        </v-tab-item>
-      </v-tabs-items>
-    </v-navigation-drawer>
-
-    <!-- Top horizontal toolbar -->
-    <v-app-bar v-if="!loadingSketch" app hide-on-scroll clipped flat :color="$vuetify.theme.dark ? '#121212' : 'white'">
-      <v-btn icon v-show="!showLeftPanel && !loadingSketch" @click="toggleLeftPanel" class="ml-n1">
+      <v-btn icon @click="toggleLeftPanel">
```

Comment:
What do you think about making the menu icon change depending on the state of the left panel. This way it is more clear what it does from my point of view. E.g. by using `mdi-menu-close` if the left panel is visible.